sentry-core/src/test/java/io/sentry/core/SentryClientTest.kt

```diff
@@ -372,6 +373,30 @@ class SentryClientTest {
         assertTrue(sentryOptions.transportGate!!.isSendingAllowed)
     }
 
+    @Test
+    fun `when scope has event processors, they should be applied`() {
+        val event = SentryEvent()
+        val scope = createScope()
+        val processor = mock<EventProcessor>()
+        scope.addEventProcessor(processor)
+
+        val sut = fixture.getSut()
+
+        sut.captureEvent(event, scope)
+        verify(processor, times(1)).process(eq(event), anyOrNull())
```

Comment:
You could remove `times(1)` here, because `Mockito.verify()` with only one parameter anyway uses `times(1)`.

Reply:
@jonesphillip good catch :) time to replace `, times(1)` for nothing hehe, thanks!

Reply:
Thanks :) I opened up a PR to get rid of all of them in your repository: https://github.com/getsentry/sentry-android/pull/211